src/flexalloc_mm.c

```diff
@@ -601,7 +632,7 @@ fla_super_read(struct xnvme_dev *dev, size_t lb_nbytes, struct fla_super **super
   if (FLA_ERR(err, "fla_xne_sync_seq_r_nbytes()"))
     goto free_super;
 
-  // TODO: Should this be placed in fla_xne_dev_sanity_check
+  // TODO: Should this be placed in fla_dev_sanity_check
```

Comment:
Since we are here should we take care of the TODO, either remove if necessary or move the code in question.

Reply:
logically this has nothing to do with ZNS  and though it has to do (very little as it is just a comment) with the ZNS patch, I do not think it merits adding this to the patch.